android/expoview/src/main/java/host/exp/exponent/ReactNativeStaticHelpers.java

```diff
@@ -33,6 +33,28 @@ public static void setBundleUrl(final String bundleUrl) {
     sBundleUrl = bundleUrl;
   }
 
+  @DoNotStrip
+  public static void reloadFromManifest(final int activityId) {
+    try {
+      Class.forName("host.exp.exponent.kernel.Kernel")
+          .getMethod("reloadVisibleExperience", int.class)
+          .invoke(null, activityId);
+    } catch(Exception e) {
+      Log.e("reloadFromManifest", "Unable to reload visible experience", e);
+    }
+  }
+
+  @DoNotStrip
+  public static String getManifestUrlForActivityId(final int activityId) {
```

Comment:
is this ever called?

Reply:
it's not. i originally planned on using it but didn't, i thought i'd leave it in as someone might want it in the future but on second thought i'll just remove it